setup.cfg

```
@@ -39,7 +39,6 @@ install_requires =
     django-simple-certmanager
     django-solo
     lxml
```

```suggestion
    lxml>=3.0
```

`resolve_entities` parameter was added in that version (with a default of `True`). From version 5.0 on the default has been changed to `'internal'`, which only resolves the entities which can be resolved within the document, without going to external sources like disk or network.

---

For reference: https://github.com/tiran/defusedxml/blob/c7445887f5e1bcea470a16f61369d29870cfcfe1/README.md#defusedxmllxml

---

the lower bound is (indirectly) set in python3-saml to 4.7.1+, but I'll apply the change anyway since explicit is better than implicit :)